### PR TITLE
Feature/more answer requests

### PR DESCRIPTION
**Pull request by FelixSFD**
Added `/answers`, `/answers/{ids}` and `/answers/{ids}/questions`

### CODE REVIEW

File: Sources/RequestsAnswers.swift [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+     - returns: The list of questions as `APIResponse<Answer>`
+     
+     - author: FelixSFD
+     */
+    public func fetchAnswers(
+        parameters: [String:String] = [:],
+        backoffBehavior: BackoffBehavior = .wait) throws -> APIResponse<Answer> {
+        
+        return try performAPIRequest(
+            "answer",
```
Comment (jnkel) [inline Sources/RequestsAnswers.swift]: Shouldn't this be `answers`?

File: Sources/RequestsAnswers.swift [hunk 2]
```diff
@@ -0,0 +1,291 @@
+//
+//  RequestsAnswers.swift
+//  SwiftStack
+//
+//  Created by Felix Deil on 14.01.17.
```
Comment (jnkel) [inline Sources/RequestsAnswers.swift]: Did you mean to include your full (?) name here?

File: Sources/RequestsAnswers.swift [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+     
+     - parameter ids: The answer IDs to fetch.
+     
+     - parameter parameters: The dictionary of parameters used in the request
+     
+     - parameter backoffBehavior: The behavior when an APIRequest has a backoff
+     
+     - returns: The list of sites as `APIResponse<Answer>`
+     
+     - author: NobodyNada
```
Comment (jnkel) [inline Sources/RequestsAnswers.swift]: The `author` is wrong for this method and the methods below it

**Review by NobodyNada — CHANGES_REQUESTED** [review]
Great!  Just a couple of little typos.

Also, when do you think we should release `1.0.0`?

File: Sources/RequestsAnswers.swift (hunk 2 above)
Comment (FelixSFD) [reply Sources/RequestsAnswers.swift]: oops. Thanks :-D

File: Sources/RequestsAnswers.swift (hunk 1 above)
Comment (FelixSFD) [reply Sources/RequestsAnswers.swift]: yep. I'll change that

**Review by NobodyNada — APPROVED** [review]

**Pull request merged**